spec/operators/pairwise-spec.js

```diff
@@ -0,0 +1,88 @@
+/* globals describe, it, expect, expectObservable, expectSubscriptions, cold, hot */
+var Rx = require('../../dist/cjs/Rx');
+var Observable = Rx.Observable;
+
+describe('Observable.prototype.pairwise()', function () {
+  it('should pairwise things', function () {
+    var e1 = hot('--a--^--b--c--d--e--f--g--|');
+    var e1subs =      '^                    !';
+    var expected =    '------v--w--x--y--z--|';
+
+    var values = {
+      v: ['b', 'c'],
+      w: ['c', 'd'],
+      x: ['d', 'e'],
+      y: ['e', 'f'],
+      z: ['f', 'g']
+    };
+
+    var source = e1.pairwise();
+
+    expectObservable(source).toBe(expected, values);
+    expectSubscriptions(e1.subscriptions).toBe(e1subs);
+  });
+
+  it('should not emit on single-element streams', function () {
+    var e1 = hot('-----^--b----|');
+    var e1subs =      '^       !';
+    var expected =    '--------|';
+
+    var values = {
+    };
+
+    var source = e1.pairwise();
+
+    expectObservable(source).toBe(expected, values);
+    expectSubscriptions(e1.subscriptions).toBe(e1subs);
+  });
+
+  it('handle mid-stream throw', function () {
```

Comment:
I know it's quite minor thing, would you able to align test case names as same as others? (starts with `should..`)